### PR TITLE
Consolidate multicluster helidon and change-placement examples

### PR DESCRIPTION
**Pull request by desagar**
# Description

Consolidate multicluster helidon and change-placement examples and also consolidate the associated acceptance tests. Added workarounds for VZ-2454 in the ATs

Fixes VZ-2597

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [x] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/multicluster/examples/helidon/helidon_example_test.go [hunk 1]
```diff
@@ -159,8 +159,69 @@ var _ = ginkgo.Describe("Multi-cluster verify hello-helidon", func() {
 	// 	})
 	// })
 
-	ginkgo.Context("Delete resources on admin cluster", func() {
-		ginkgo.It("Delete all the things", func() {
+	ginkgo.Context("Change Placement of app to Admin Cluster", func() {
```
Comment (desagar) [inline tests/e2e/multicluster/examples/helidon/helidon_example_test.go:162]: this section is copied almost verbatim from the change placement test, except for the call to the ChangePlacement functions, since the names of those functions have changed.

File: examples/multicluster/hello-helidon/README.md [hunk 2]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
    $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl get MultiClusterApplicationConfiguration -n hello-helidon
    ```
+## Changing the placement of the application to a different cluster
+
+By default, the application will be placed in the managed cluster called `managed1`. You can change the application
+placement to be in a different cluster, which can be the admin cluster or a different managed cluster. In this example,
+we will change the placement of the application to be in the admin cluster, by patching the multicluster resources
+as follows.
+
+1. Specify the change placement patch file for changing the placement to admin cluster. We will use this environment
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:62]: to admin cluster -> to the admin cluster

File: examples/multicluster/hello-helidon/README.md [hunk 3]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
+   $ export CHANGE_PLACEMENT_PATCH_FILE="patch-change-placement-to-admin.yaml"
+   ```
+
+1. Patch the `hello-helidon` multicluster resources to change their placement.
+   ```shell
+   $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl patch mcappconf hello-helidon-appconf -n hello-helidon --type merge --patch "$(cat $CHANGE_PLACEMENT_PATCH_FILE)"
+   $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl patch mccomp hello-helidon-component -n hello-helidon --type merge --patch "$(cat $CHANGE_PLACEMENT_PATCH_FILE)"
+   ```
+1. View the multicluster resources to see that the placement has changed to be in a different cluster. Placement in the
+   admin cluster is indicated by a cluster name of `local`.
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:75]: name of `local`. -> named `local`.

File: examples/multicluster/hello-helidon/README.md [hunk 4]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
+1. Patch the VerrazzanoProject to change its placement.
+   ```shell
+   $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl patch vp hello-helidon -n verrazzano-mc --type merge --patch "$(cat $CHANGE_PLACEMENT_PATCH_FILE)"
+   ```
+1. Wait for the application to be ready on the admin cluster.
+   ```shell
+   $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl wait --for=condition=Ready pods --all -n hello-helidon --timeout=300s
+   ```
+   **Note:** If you are returning the application to the managed cluster, then instead wait for application to be ready
+   again on the managed cluster.
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:89]: managed cluster, then instead wait for application  -> managed cluster, then instead, wait for the application

File: examples/multicluster/hello-helidon/README.md [hunk 5]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
+   ```shell
+   $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl wait --for=condition=Ready pods --all -n hello-helidon --timeout=300s
+   ```
+
+You can now test the example application running in the cluster where you have now placed it.
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:94]: You can now test -> Now, you can now test

File: examples/multicluster/hello-helidon/README.md [hunk 6]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
+
+To return the application back to the managed cluster named `managed1`, simply use the patch file provided for that
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:96]: the application back to the managed cluster -> the application to the managed cluster

File: examples/multicluster/hello-helidon/README.md [hunk 7]
```diff
@@ -43,20 +43,67 @@ Follow the [instructions](../../hello-helidon/README.md/#testing-the-example-app
...
+purpose as the CHANGE_PLACEMENT_PATCH_FILE as shown below, then repeat the numbered steps above to complete the process.
```
Comment (rosemarymarano) [inline examples/multicluster/hello-helidon/README.md:97]: CHANGE_PLACEMENT_PATCH_FILE as shown below -> CHANGE_PLACEMENT_PATCH_FILE shown below

**Review by rosemarymarano — COMMENTED** [review]
A few minor edits.

File: examples/multicluster/hello-helidon/README.md (hunk 3 above)
Comment (desagar) [reply examples/multicluster/hello-helidon/README.md:75]: I am not sure how to reword it to use "named". What I am trying to say here is that in order to have placement in the admin cluster, the cluster name should be specified as "local". 

I could just say "The admin cluster is named `local`." but that doesn't make the connection to how you specify it in placement. Another alternative is to say "The cluster name `local` indicates placement in the admin cluster".

What do you think?

Comment (rosemarymarano) [reply examples/multicluster/hello-helidon/README.md:75]: yes, I get it now. I think this is good, "The cluster name, `local`, indicates placement in the admin cluster".

File: examples/multicluster/hello-helidon/README.md (hunk 7 above)
Comment (desagar) [reply examples/multicluster/hello-helidon/README.md:97]: I think I'll reword this a bit more, since I see that it is missing some information anyway. 

I plan to reword that part of the sentence as "simply use the patch file provided for that purpose as the value of the CHANGE_PLACEMENT_PATCH_FILE environment variable, then repeat the numbered steps above to complete the process."

Let me know if you're ok with that.

Comment (rosemarymarano) [reply examples/multicluster/hello-helidon/README.md:97]: We usually put the phrase first, then the action, hence... "As the value of the `CHANGE_PLACEMENT_PATCH_FILE` environment variable, simply use the patch file provided for that purpose, then repeat the numbered steps above to complete the process."

Comment (desagar) [reply examples/multicluster/hello-helidon/README.md:97]: What do you think we should do with the first part of that sentence i.e. "To return the application back to the managed cluster named managed1, "

(that is the "purpose" that is referred to later in that sentence).

The main point of this paragraph is returning the application to the managed cluster. How about 

"To return the application to the managed cluster named `managed1`, set the value of the CHANGE_PLACEMENT_PATCH_FILE environment variable to the patch file provided for that purpose, then repeat the numbered steps above to complete the process."

Comment (desagar) [reply examples/multicluster/hello-helidon/README.md]: I changed the sentence as I suggested above. Let me know if you want it to be different.

**Review by mgianatagh — APPROVED** [review]

File: examples/multicluster/hello-helidon/README.md (hunk 7 above)
Comment (rosemarymarano) [reply examples/multicluster/hello-helidon/README.md]: Looks good.
